README.md

```diff
@@ -82,7 +82,7 @@ sudo apt build-dep pop-gnome-shell-theme
 
 Or, use this to install all dependencies in one go:
 ```
-sudo apt install libsass0 sassc inkscape optipng libglib2.0-dev-bin
+sudo apt install libsass1 sassc inkscape optipng libglib2.0-dev-bin
```

Comment:
As mentioned in the build-deps PR, we should probably remove this explicit `libsass` dependency. `sassc` will pull in whichever version is current for the user's distro, and won't break it for bionic vs. cosmic.

Reply:
Removed!